### ISSUE TITLE
how do the tools get updated

### ISSUE THREAD

**Issue by samalloing** [openpreserve/ddhn-forge#19]
Hi @carlwilson (cc @aniamolenda)

How can the tools (jhove, verapdf,...) be updated without reloading vagrant?

Sam

**Comment by carlwilson** [comment]
Short answer is to use `vagrant provision` but using the right portion of the playbook (just a single tool install) would also work. There needs to be good documentation here.

**Comment by samalloing** [comment]
That is true, but user need to change the config file. I don't know if this is user friendly enough for the intended audience (not that I know who that is ;-)). There is no repository to install this software from I guess (like rpm based repositories)

**Comment by carlwilson** [comment]
While we are keen to improve the documentation here to allow users to do this, we're unlikely to change the mechanism itself. We lack the resources to support custom distributions and the tool is aimed at technically inexperienced users who can't configure virtual environments.
